test/units/testsuite-48.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -ex
+
+test_rule="/run/udev/rules.d/48-test.rules"
+
+setup() {
+    mkdir -p "${test_rule%/*}"
+    cp -f /etc/udev/udev.conf /etc/udev/udev.conf.bckp
+    echo 'KERNEL=="lo", SUBSYSTEM=="net", PROGRAM="/usr/bin/sleep 2"' > "${test_rule}"
+    echo "event_timeout=1" >> /etc/udev/udev.conf
+    echo "timeout_signal=SIGABRT" >> /etc/udev/udev.conf
+
+    systemctl restart systemd-udevd.service
+}
+
+teardown() {
+    set +e
+
+    mv -f /etc/udev/udev.conf.bckp /etc/udev/udev.conf
+    rm -f "$test_rule"
+    systemctl restart systemd-udevd.service
+}
+
+run_test() {
+    since="$(date +%T)"
+
+    echo add > /sys/class/net/lo/uevent
+    sleep 10
+
+    coredumpctl --since "$since" --no-legend --no-pager | grep /usr/lib/systemd/systemd-udevd
+}
+
+trap teardown EXIT
+
+setup
+run_test
```

Comment:
Ah, one more (and hopefully last) thing to make this work:

```bash
echo OK > /testok

return 0
```
should be there as well, so the default `check_results_qemu/check_results_nspawn` functions pass:

https://github.com/systemd/systemd/blob/1c36fb6ad446131afea2af9596a935466e3066bd/test/test-functions#L810-L814

Reply:
Given the intermediate results it looks like the test is finally passing, thanks!